charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -397,6 +397,12 @@ defaultCRConfig:
     logsCollection:
       enabled: false
       includeCollectorLogs: true
+      # If includeCollectorLogs is set to false, below list of container names
+      # used in collector would be used to generate exclusion rules
+      collectorContainerNames:
+        - otc-container
+        # - init-nginx
+        # - additional-container
```

Comment:
I dont like the precedence this sets. We have historically not allowed the preset logic to be configurable. I'd prefer the preset make an assumption for the collector container name based on the chart's default. @jaronoff97 what do you think?